shell/src/Luna/Shell/Option.hs

```diff
@@ -31,14 +31,25 @@ lunaCommand = Command.Exec <$> Options.hsubparser
     (  Options.command "run" (Options.info run
         (Options.progDesc "Execute a luna package, or standalone file."))
     <> Options.command "init" (Options.info init
-        (Options.progDesc "Initialise a new luna package.")))
+        (Options.progDesc "Initialise a new luna package."))
+    <> Options.command "document" (Options.info document
+        (Options.progDesc "Print documentation in JSON format.")))
 
 run :: Parser Command
 run = Command.Run <$> (Command.RunOpts
     <$> Options.strOption (Options.long "target"
         <> Options.metavar "FILE/FOLDER" <> Options.value ""
         <> Options.help "Execute FILE/FOLDER in interpreted mode."))
 
+document :: Parser Command
+document = Command.Document <$> (Command.DocumentOpts
+    <$> Options.strOption (Options.long "target"
+        <> Options.metavar "FOLDER" <> Options.value ""
+        <> Options.help "Execute FOLDER in interpreted mode.")
```

Comment:
I think it can be slightly misleading, as we are not executing the project here. What about "Generate documentation for PROJECT_PATH"?

Reply:
With the accompanying metavar change to `PROJECT`, I think just 'Generate documentation for PROJECT' would work.